pkg/kudoctl/setup/setup.go

```diff
@@ -0,0 +1,115 @@
+package setup
```

Comment:
This is now the main entry point for initialization of the cluster - At some point this should also provide functionality to validate the installation, upgrade the KUDO installation, etc.